README.md

```diff
@@ -1,79 +1,41 @@
-# ![Islandora 8](https://camo.githubusercontent.com/738dd7cbd90a3ef06b9bb55a4cf5ed385a048fd4/687474703a2f2f69736c616e646f72612e63612f73697465732f64656661756c742f66696c65732f696d616765732f6c6f6273746572434c41572e706e67) Islandora 8 Documentation
+# Islandora 8 Documentation
+![Islandora 8](https://camo.githubusercontent.com/738dd7cbd90a3ef06b9bb55a4cf5ed385a048fd4/687474703a2f2f69736c616e646f72612e63612f73697465732f64656661756c742f66696c65732f696d616765732f6c6f6273746572434c41572e706e67)
 
-## About this repository (github.com/Islandora/documentation)
+[Read the documentation online](https://islandora.github.io/documentation/)
 
-This repository contains the user, developer, and administrator documentation for the [Islandora project](https://islandora.ca/), for versions 8 and above. This documentation is available to read at https://islandora.github.io/documentation/, where it is automatically deployed on commit from the source code at https://github.com/Islandora/documentation. Documentation for Islandora 7.x and previous versions is available at https://wiki.lyrasis.org/display/ISLANDORA/. 
+## About the Documentation
```

Comment:
I can't seem to manage a pull request against your fork, but I do have some wording here to roll in @eldonquijote 's suggestion:

`## About the Documentation

This repository has three primary uses:

- it contains source code for the user, developer, and administrator documentation for versions 8.x and above of the [Islandora project](https://islandora.ca/). It is written in mkdocs and can be viewed on [our Github Pages site](https://islandora.github.io/documentation/).
- its [Wiki](https://github.com/Islandora/documentation/wiki) contains the minutes for Islandora 8 Tech calls and User calls.
- it hosts the central [issue queue]((https://github.com/Islandora/documentation/issues) for the entire Islandora 8 project.`

Reply:
I made a couple changes for brevity - does it still work?